packages/element-app/src/components/Snackbar/MySnackbarContentWrapper.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import clsx from 'clsx';
+
+import CheckCircleIcon from '@material-ui/icons/CheckCircle';
+import ErrorIcon from '@material-ui/icons/Error';
+import InfoIcon from '@material-ui/icons/Info';
+import CloseIcon from '@material-ui/icons/Close';
+
+import IconButton from '@material-ui/core/IconButton';
+
+import SnackbarContent from '@material-ui/core/SnackbarContent';
+import WarningIcon from '@material-ui/icons/Warning';
+import { makeStyles } from '@material-ui/core/styles';
+
+// import { red, yellow, green, blue } from '@material-ui/core/colors';
```

Comment:
nit: cleanup

Reply:
I opted to comment this for the case where we support non branded options.